docs/Splashkit/Applications/Arcade Machines/Arcade Machine Setup/AddGameOverNetwork.md

```diff
@@ -0,0 +1,116 @@
+
+### Game Files
+This guide assumes that you are transferring **compiled** game binaries to the Arcade Machine. The Arcade Machine uses a Raspberry Pi, which has an ARM processor. When compiling your games make sure you **cross compile** for arm32. If your game is packaged as a .deb file you can install it connecting to the Arcade Machine via ssh (```ssh deakin@10.42.0.1```) and then running ```sudo dpkg -i myGame.deb``` to install it locally.
+
+### The "authenticity of host" warning message
+If this is the first time that you are connecting to the Arcade Machine you might see a prompt similar to the one below saying that authenticity of the host can't be established. **In this specific scenario** it's safe to type "yes" and continue:
+
+```
+user@host:~$ ssh user@arcademachine.local
+
+The authenticity of host '<host>' can't be established.
+ECDSA key fingerprint is    SHA256:<SomeHash>.
+Are you sure you want to continue connecting (yes/no)?
+```
+
+
+## Connecting To Arcade Machine WiFi Access Point
+Find the SSID of the WiFi access point corresponding to the Arcade Machine you want to copy games to. When prompted enter the WiFi password. By default this is ```GamesAreFun```. 
+
+The Arcade Machine has no [DHCP](https://en.wikipedia.org/wiki/Dynamic_Host_Configuration_Protocol) service, so you will need to manually set a static IP address once you have connected to the network. You will need the following network settings:
+
+The Arcade Machine subnet is ```10.42.0.1 /24``` Your network configuration should therefore look like:
+```
+IP Address:  10.42.0.2
+Subnet Mask: 255.255.255.0 
+Gateway:     10.42.0.1
+```
+
+**Note:** If another student is concurrently accessing the Arcade Machine you will need to choose another IP Address, or you won't be able to use the network!
+
+Verify connectivity by pinging the Arcade Machine's IP address:
+```
+user@host:~$ ping 10.42.0.1
+PING 10.42.0.1 (10.42.0.1) 56(84) bytes of data.
+64 bytes from 10.42.0.1: icmp_seq=1 ttl=58 time=43.0 ms
+64 bytes from 10.42.0.1: icmp_seq=2 ttl=58 time=47.0 ms
+64 bytes from 10.42.0.1: icmp_seq=3 ttl=58 time=40.8 ms
+64 bytes from 10.42.0.1: icmp_seq=4 ttl=58 time=41.1 ms
+64 bytes from 10.42.0.1: icmp_seq=5 ttl=58 time=40.6 ms
+^C
+--- 10.42.0.1 ping statistics ---
+5 packets transmitted, 5 received, 0% packet loss, time 4001ms
+user@host:~$ 
+```
+
+## Secure Copy Protocol (SCP)
+The simplest method of transferring files to the Arcade Machine is to use [SCP](https://en.wikipedia.org/wiki/Secure_copy_protocol). The basic syntax of this command is:
+
+```scp [options] [[user@]host1:]source [[user@]host2:]destination```
+
+In this scenario the source is your computer and the destination is the Arcade Machine. SCP requires you to authenticate with the destination host in order to send files over the network. This means that you need to provide authentication credentials for the **Arcade Machine**. By default the SSH user's credentials are ```deakin``` and ```qwerty```. 
```

Comment:
When should one use the deakin user vs the qwerty user? Also what's the qwerty user used for?

Reply:
Fair call. I've modified the language to be a bit more explicit.